tests/foreman/cli/test_contentview.py

```diff
@@ -5323,7 +5322,26 @@ def test_positive_arbitrary_file_repo_promotion(self):
         :expectedresults: Check arbitrary files from FR is available on
             environment
 
-        :CaseAutomation: notautomated
+        :CaseAutomation: automated
 
         :CaseLevel: Integration
         """
+        org = ContentViewFileRepoTestCase.org
```

Comment:
```suggestion

```
Looks not require as self.org is an object and can directly call.

Reply:
Right, Same has been taken care.